src/components/molecules/Chatbox/Chatbox.module.scss

```diff
@@ -0,0 +1,23 @@
+@use "scss/attendee/space";
+
+.chatboxContainer {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.chatboxMessages {
+  flex-grow: 1;
+  // Start tiny and grow so that the send box is always visible
+  flex-basis: 1px;
+  overflow-y: scroll;
+  padding: space.empty(1.5);
+}
+
+.messageScroller {
+  display: flex;
+  flex-direction: column-reverse;
+  // Required to prevent inline styles from the infinite scroller creating
+  // a horizontal scrollbar
+  overflow-x: hidden !important;
```

Comment:
[code smell] Still very important to avoid `!important` and if it really can't be helped, maybe a comment of why there are inline styles

Reply:
I'll expand on the reasoning for this.

Reply:
I assumed it was the library, just wasn't sure